src/components/enterprise-user-subsidy/data/hooks/useSubscriptions.js

```diff
@@ -27,7 +28,15 @@ function useSubscriptions({
 
   useEffect(
     () => {
-      setSubscriptionPlan(subscriptionLicense?.subscriptionPlan);
+      if (subscriptionLicense?.subscriptionPlan) {
+        setSubscriptionPlan({
+          ...subscriptionLicense.subscriptionPlan,
+          isCurrent: hasValidStartExpirationDates({
```

Comment:
question: why `isCurrent` here and `available` for codes?

Reply:
@iloveagent57 Enterprise offers API returns an `is_current` (`isCurrent` when camelCased). I opted to use `available` for codes because the related `couponsOverview` data uses `available` to mark whether the parent coupon is current/open. It seemed to make sense to align an `available` field on coupon codes with the `available` field on the associated coupon tied to the code. For subscriptions, there was no previous notion of `isCurrent` or `available` so opted for the more preferable `isCurrent` there.